instrumentation/jmx-metrics/javaagent/camel.md

```diff
@@ -0,0 +1,51 @@
+| camel.context.exchange                         | Counter       | context, camelVersion                  | Indicates the total number of exchanges, passed or failed, processed since context start-up or the last reset operation.                                                                                                                                          |
+| camel.context.exchange_completed               | Counter       | context, camelVersion                  | Indicates the total number of exchanges processed successfully since context start-up or the last reset operation.                                                                                                                                                |
+| camel.context.exchange_failed                  | Counter       | context, camelVersion                  | Indicates the total number of exchanges that failed to process since context start-up or the last reset operation.                                                                                                                                                |
+| camel.context.exchange_failed_handled          | Counter       | context, camelVersion                  | Indicates the number of exchanges failed and handled by an ExceptionHandler in the context.                                                                                                                                                                       |
+| camel.context.exchange_inflight                | UpDownCounter | context, camelVersion                  | Indicates the number of exchanges currently transiting the context.                                                                                                                                                                                               |
+| camel.context.exchange_processing_delta_time   | Gauge         | context, camelVersion                  | Indicates the difference, in milliseconds, of the Processing Time of the last two exchanges transited the context.                                                                                                                                                |
+| camel.context.exchange_processing_last_time    | Gauge         | context, camelVersion                  | Indicates the time, in milliseconds, it took to process the last exchange.                                                                                                                                                                                        |
+| camel.context.exchange_processing_max_time     | Gauge         | context, camelVersion                  | Indicates the longest time, in milliseconds, to process an exchange since context start-up or the last reset operation.                                                                                                                                           |
```

Comment:
unfortunately I think switching from `.` to `_` goes against recommendation from the semconv SIG in https://github.com/open-telemetry/semantic-conventions/issues/1118#issuecomment-2173803006

> we need to update attribute naming guidance and suggest to use `.` when it makes sense